### ISSUE TITLE
Feature Request: option to maintain new way of hack without animation

### ISSUE THREAD

**Issue by sleep-walker** [broo2s/ingress-apk-mod#6]
Workflow of new way of hacking introduced in 1.30.2 is not that bad.

Please, add option to disable animation of the hack, but keep the behaviour of no big window but small notification as it is.

1] click hack
2] get back from portal view to map view
(no animation here)
3] show results of hacking

**Comment by broo2s** [comment]
Yes, I was thinking about it, because new hack summary window is nice. For now I did what was the easiest.
